### PR TITLE
Feature/hacpp 7 tooltip

### PR DESCRIPTION
**Pull request by beloglazov91**

### CODE REVIEW

File: src/components/form/Tooltip/tooltip.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  function handleTooltip(e) {
+    e.preventDefault();
+
+    const tooltip = e.target.closest('.tooltip');
+    const body = tooltip.querySelector('.tooltip__body');
+
+    closeAllTooltips(body);
+
+    console.log(tooltip.getBoundingClientRect().left);
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.js:23]: trailing console log

File: src/components/form/Tooltip/tooltip.scss [hunk 2]
```diff
@@ -0,0 +1,102 @@
+@import "src/lib.scss";
+
+.tooltip {
+  display: inline-block;
+  position: relative;
+  vertical-align: middle;
+
+  &__question-mark {
+    cursor: pointer;
+    height: 36px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:10]: can you convert to rems please?

File: src/components/form/Tooltip/tooltip.scss [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    position: relative;
+    width: 36px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:12]: here too please

File: src/components/form/Tooltip/tooltip.scss [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    z-index: 2;
+
+    &::before {
+      content: "";
+      background-image: url("./assets/question.svg");
+      display: inline-block;
+      width: 36px;
+      height: 36px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:20]: and these 2 too please

File: src/components/form/Tooltip/tooltip.scss [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  &__close {
+    cursor: pointer;
+    display: block;
+    height: 11px;
+    position: absolute;
+    right: 0.5rem;
+    top: 0.5rem;
+    width: 11px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:32]: also here (and the width above)

File: src/components/form/Tooltip/tooltip.scss [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+    &::before {
+      content: "";
+      background-image: url("./assets/close.svg");
+      display: block;
+      width: 11px;
+      height: 11px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:39]: and here

File: src/components/form/Tooltip/tooltip.scss [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  &__body {
+    background: var(--fsa_online-grey);
+    border: 3px solid var(--fsa-primary_dark-green);
+    display: none;
+    font-size: 1rem;
+    position: absolute;
+    right: 1rem;
+    top: 59px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:51]: and the top too please

File: src/components/form/Tooltip/tooltip.scss [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+    &::before {
+      background: var(--fsa_online-grey);
+      border-right: 3px solid var(--fsa-primary_dark-green);
+      content: "";
+      position: absolute;
+      right: -3px;
+      top: -15px;
+      height: 15px;
+      width: 15px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:73]: and all of these please (except the border)

File: src/components/form/Tooltip/tooltip.scss [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    &::after {
+      background: var(--fsa-primary_white);
+      border-right: 3px solid var(--fsa-primary_dark-green);
+      content: "";
+      position: absolute;
+      right: 1px;
+      top: -27px;
+      height: 22px;
+      transform: rotate(45deg);
+      width: 22px;
```
Comment (c-ancia) [inline src/components/form/Tooltip/tooltip.scss:85]: and also these please (except the border)

**Review by c-ancia — CHANGES_REQUESTED** [review]

**Comment by beloglazov91** [comment]
@c-ancia All amends are here :)

**Pull request merged**